components/klab.component.random/src/main/java/org/integratedmodelling/random/adapters/RecreationIDBRuntimeEnvironment.java

```diff
@@ -0,0 +1,165 @@
+package org.integratedmodelling.random.adapters;
+
+import kong.unirest.GetRequest;
+import kong.unirest.HttpResponse;
+import kong.unirest.Unirest;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Objects;
+import java.util.logging.Logger;
+
+import org.integratedmodelling.klab.exceptions.KlabException;
+
+public class RecreationIDBRuntimeEnvironment {
+
+    public enum HTTPStatusCode {
+        OK(200), Unauthorized(401);
```

Comment:
Why only consider "Success" or "Unauthorized", what about `429` (too many requests) and other `5xx`.
Are not all this usual values part of `kong.unirest`? 🤔

Reply:
Checking the documentation (OpenAPI) the only possible reponse codes are 200 or 401 🤷🏾‍♂️